src/test/java/com/passbook/sparkeighteen/service/UserServiceTest.java

```diff
@@ -162,5 +162,34 @@ public void inValidPayload_passwordMismatch_loginUnSuccessful() throws Exception
 
     }
 
+    @Test
```

Comment:
What will happen if we pass null?

Reply:
even you enter null user id it will first check for that user exist or not then further logic will perform, if it is there then profile will delete else unsuccessful

Reply:
it will only work if user found with that userId